artiq/frontend/artiq_sinara_tester.py

```diff
@@ -340,6 +343,63 @@ def mirny_rf_switch_wave(self, channels):
             for channel in channels:
                 channel.pulse(100*ms)
                 delay(100*ms)
+    @kernel
+    def init_almazny(self, almazny):
+        self.core.break_realtime()
+        almazny.init()
+
+    @kernel
+    def almazny_set_attenuators(self, almazny, ch, atts):
+        self.core.break_realtime()
+        almazny.set_att_mu(ch+1, atts, output_on=True)
+
+    @kernel
+    def almazny_set_attenuators_db(self, almazny, ch, atts):
+        self.core.break_realtime()
+        mu = almazny.att_to_mu(atts)
+        almazny.set_att_mu(ch+1, mu, output_on=True)
+
+    @kernel
+    def almazny_rf_toggle(self, almazny, ch, on):
+        self.core.break_realtime()
+        almazny.set_att_mu(ch+1, 63, on)
+
+    def test_almaznys(self):
+        print("*** Testing Almaznys.")
+        for name, almazny in sorted(self.almaznys.items(), key=lambda x: x[0]):
+            print(name + "...")
+            self.init_almazny(almazny)
+            print("Testing attenuators. Frequencies:")
+            for card_n, channels in enumerate(chunker(self.mirnies, 4)):
+                for channel_n, (channel_name, channel_dev) in enumerate(channels):
+                    frequency = 1000*(card_n + 2) + channel_n * 100
+                    print("{}\t{}MHz (double is {}MHz)".format(channel_name, frequency, frequency*2))
```

Comment:
Only the doubled frequency matters here I think?
The other output should be tested during the Mirny test.